ChromeDriverAmazonMusic.py

```diff
@@ -8,16 +8,15 @@
 from selenium.webdriver.common.by import By
 from selenium.webdriver.support.ui import WebDriverWait
 from selenium.webdriver.support import expected_conditions as EC
+from FuzzyMatcher import *
```

Comment:
Import everything from a package will affect the performance. Import only the required things.

Reply:
Hmmm.